packages/microsite/README.md

```diff
@@ -56,3 +56,31 @@ project/
 - [Lukas Bombach](https://twitter.com/luke_schmuke), [The case of partial hydration (with Next and Preact)](https://medium.com/@luke_schmuke/how-we-achieved-the-best-web-performance-with-partial-hydration-20fab9c808d5)
 - [Jason Miller](https://twitter.com/_developit) and [Addy Osmani](https://twitter.com/addyosmani), [Rendering on the Web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)
 - [Poimandres](https://github.com/pmndrs), [`valtio`](https://github.com/pmndrs/valtio) for inspiring `microsite/global`
+
+## Contributing
+
+This is a monorepo, set up with [Lerna](https://github.com/lerna/lerna), you should start by familiarizing yourself with its commands, as it's a little bit different from standard Node/NPM workflows. The most important thing to remember is that commands should generally be run from the monorepo root directory.
+
+### Initial dev setup
+
+1. Install [NVM](https://github.com/nvm-sh/nvm/) if you haven't already, to ensure you're using the Node/NPM version specified by the project.
```

Comment:
I'm actually using [Volta](https://volta.sh/) to manage node versions—it's much faster than `nvm` and automatically switches everything for you. I can update these notes, but thanks so much for starting them!

Reply:
Ah I'll go ahead and update that and use it for my dev setup too. Thanks! I've had more problems with NVM than I'd like. If you don't mind, I will probably also include a warning about `curl ... | bash`, because [it's insecure](https://www.idontplaydarts.com/2016/04/detecting-curl-pipe-bash-server-side/).